docker/Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM jenkins/jenkins:2.492.3-lts-jdk21
 LABEL maintainer="OpenSearch"
 ENV JAVA_OPTS -Djenkins.install.runSetupWizard=false
 COPY plugins.txt plugins.txt
+RUN curl -SfL https://raw.githubusercontent.com/opensearch-project/opensearch-build/refs/heads/main/docker/ci/config/op-setup.sh -o op-setup.sh && bash op-setup.sh && rm -v op-setup.sh
```

Comment:
How about adding it in userdata?

Reply:
I am confused, basically I already add the op-cli installation on userdata.
Which only install op-cli on the EC2 instance not the docker container.
Therefore I update the dockerfile to include the cli in docker image. Anything missing?